### ISSUE TITLE
Multiprocess nmap problem.

### ISSUE THREAD

**Issue by geoleven** [savon-noir/python-libnmap#88]
Running os: latest update Linux Mint
Nmap version: 7.01
Python version: 2.7.12
Libnmap 0.7.0
Kernel: 4.13.0-38 x64

When having a process with threds, thread pools and many instances of nmapprocess some of them fail with no reason. Return code is 0 (not as expected) and stdout is totally empty but not None if I am mistaken. 

Moreover, alhtough I am not sure it is releated, if I run the program for a extensive period of time it will get me segmentation fault and stop (no memory usage should occur by the program which is not garbage collectible as soon as possible). I am trying to find the error with gdb and cpython extensions but without much luck.

**Comment by savon-noir** [comment]
Hello @geoleven  Sorry for late reply. This might be resolved by https://github.com/savon-noir/python-libnmap/pull/79

I'll merge it in next release: 0.7.1

**Comment by savon-noir** [comment]
fixed by new release [v0.7.2](https://github.com/savon-noir/python-libnmap/releases/tag/v0.7.2). 

Package is also available in pip:

```
ronald@brouette:~/dev$ pip3 install python-libnmap==0.7.2
Collecting python-libnmap==0.7.2
  Downloading python-libnmap-0.7.2.tar.gz (36 kB)
...
...
Successfully installed python-libnmap-0.7.2
ronald@brouette:~/dev$ 
```
